katello_certs_tools/sslToolConfig.py

```diff
@@ -647,15 +635,15 @@ def updateLegacy(self, newdir=None, verbosity=1):
 
         try:
             rotated = rotateFile(filepath=self.filename, verbosity=verbosity)
-            if verbosity>=0 and rotated:
-                print "Rotated: %s --> %s" % (os.path.basename(self.filename),
-                                              os.path.basename(rotated))
+            if verbosity >= 0 and rotated:
+                print("Rotated: %s --> %s" % (os.path.basename(self.filename),
+                                              os.path.basename(rotated)))
         except ValueError:
             pass
         fo = open(self.filename, 'w')
         fo.write(newfile)
         fo.close()
-        os.chmod(self.filename, 0600)
+        os.chmod(self.filename, '0600')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)